src/html/authorize.html.template

```diff
@@ -44,17 +44,17 @@
     <%= renderTemplate('_js_base') %>
     <%= renderTemplate('_preload_roboto') %>
 
-    <script type="module" crossorigin="use-credentials">
-      import "<%= latestPageJS %>";
+    <script crossorigin="use-credentials">
```

Comment:
Do we need this PR any more? https://github.com/home-assistant/frontend/pull/7318

Reply:
Nope that's all in this one